students/Sally_Shen/lesson3/mailroom.py

```diff
@@ -0,0 +1,96 @@
+
+
+'''choms'''
+
+import sys
+
+print("in mailroom")
+'''Writing first name is easier for testing.'''
+donors ={"Fred": [100, 200, 50], "Barney": [100, 50, 600], "William": [200, 50, 500],
+         "Peddle": [300, 40, 200]}
+
+
+def thank_you():
+    response = input("Type your donor name, if don't know, type list to pick")
+    '''If the user types list show them a list of the donor names and re-prompt'''
+    if response == "list":
+        for key in donors.keys():
+            print(key)
+        response = input("please type your donor name")
+
+    if response not in donors.keys():
+        donors[response] = []
+
+    newDonation = input("how much to donate?")
```

Comment:
Don't forget that once you take the input of the donation amount, you need to convert it to an integer. Otherwise it reads in as a string and the next line throws an error!